src/workspaces/cookiefactoryv2/CookieFactoryDemo/src/lib/core/graph/graph.ts

```diff
@@ -184,17 +184,17 @@ export function createGraph<EntityData>(
   // public api
 
   function center(node?: NodeCollection | NodeSingular | string) {
-    if (isNil(node)) {
-      cy.center();
-    } else {
-      if (isString(node)) {
-        node = cy.nodes(`#${node}`);
-      }
-
-      if (!nodesInView(node)) {
-        cy.center(node);
-      }
+    if (isString(node)) {
+      node = cy.nodes(`#${node}`);
     }
+
+    cy.center(node);
+  }
+
+  function centerHorizontally() {
```

Comment:
just curious if there could be a need for "centerVertically" later and to confirm if there's any assumption on the panel layout this is based on (e.g. if graph panel today takes up more horizontal space than vertical space)

Reply:
currently the panel view layout is horizontal and content only needs to be centered horizontally. a `centerVertically` method will be needed when panels are more dynamic and will be stacked in columns.